### PR TITLE
[Consignments] Pre-fixes and select the newly taken camera image to the photo selector

### PR DESCRIPTION
**Pull request by orta**
Verifies the photo is data we expect, then adds it at the beginning and adds the photos uri to the selected objects.

### CODE REVIEW

File: Example/Emission.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -441,7 +441,7 @@
 						CreatedOnToolsVersion = 7.2.1;
 						DevelopmentTeam = 23KMWZ572J;
 						LastSwiftMigration = 0830;
-						ProvisioningStyle = Manual;
+						ProvisioningStyle = Automatic;
```
Comment (sarahscott) [inline Example/Emission.xcodeproj/project.pbxproj]: Are these changes intentional?

**Comment by orta** [comment]
Nope -  great catch!

File: src/lib/Components/Consignments/Screens/SelectFromPhotoLibrary.tsx [hunk 2]
```diff
@@ -80,19 +80,26 @@ export default class SelectFromPhotoLibrary extends React.Component<Props, State
...
+        // Update selection
+        this.state.selection.add(photo.image.uri)
+        this.setState({
+          selection: this.state.selection,
+          // An item in cameraImage is a subset of the photo that we pass in,
+          // so we `as any` to avoid a compiler error
+          cameraImages: data.edges.map(e => e.node).concat(this.state.cameraImages as any),
+        })
+      } else {
+        console.log("CameraRoll: Did not recieve a photo from call to getPhotos")
```
Comment (sarahscott) [inline src/lib/Components/Consignments/Screens/SelectFromPhotoLibrary.tsx]: receive 😉

File: Example/Emission.xcodeproj/project.pbxproj (hunk 1 above)
Comment (orta) [reply Example/Emission.xcodeproj/project.pbxproj]: and they're gone 👍

**Comment by sarahscott** [comment]
lgtm on 🍏 !
